### PR TITLE
Updated SWHS IMs from Manual

### LINKED ISSUE #1064: SWHS: Difference b/w Stable and Manual: Missing Information in IM1
Issue #523 has far too much going on to keep track of all of the details. This issue (and several others) highlights one specific difference between Stable and Manual.

**1.** Assumptions associated with the input and output of IM1
Stable:
image
Manual:
image

**2.** Description is missing associated assumptions and constraints 
Stable: 
image
Manual:
image

### LINKED ISSUE #1069: SWHS: Difference b/w Stable and Manual: Match output of IM2
Issue #523 has far too much going on to keep track of all of the details. This issue (and several others) highlights one specific difference between Stable and Manual.

- Missing information for the output of IM2 (non-trivial)
Stable:
image
Manual:
image

IM2 has been updated in the stable (Drasil) version:

im2

Unfortunately the updates still don't entirely match the manual versions.  The following changes still need to be made:
* The Equation doesn't match the manual version; it should have 3 cases (not 4)
* The note should contain the following bulleted list (ideally displayed as a bulleted list for ease of reading/verifying):
  - T_W is defined by (IM: eBalanceOnWtr).
  - The input constraint Tinit ≤ T_P^melt comes from (A: PCM-Initially-Solid).
  - The temperature remains constant at TmeltP, even with the heating (or cool-ing), until the phase change has occurred for all of the material; that is as long as 0 < ϕ < 1.   ϕ (from DD: melt_frac ) is determined as part of the heat energy in the PCM, as given in (IM: heatEInPCM).
  - tau_P^S is calculated in DD? (needs to be added)
  - tau_P^L is calculated in DD? (needs to be added)

### LINKED ISSUE #1487: SWHS: Equalities in IM Constraints
_Similar to #1485_
In the manual version, constraints are inclusive

image

(The one exception is this constraint in IM2 and IM4): 
image

While they are exclusive in Drasil:

image

The manual version is correct and Drasil should be updated.

### PR DESCRIPTION
**Pull request by samm82**
Updated the IMs of SWHS to match the manual versions. This PR closes #1069, closes #1487, and closes #1064, other than the discussion about bullet lists in the Notes section.

### CODE REVIEW

File: code/drasil-example/Drasil/SWHS/IMods.hs [hunk 1]
```diff
@@ -210,25 +204,23 @@ eBalanceOnPCM :: InstanceModel
...
   [sy temp_init $< sy temp_melt_P] (qw temp_PCM)
-   [0 $< sy time $< sy time_final] [makeCite koothoor2013] eBalanceOnPCMDeriv 
-   "eBalanceOnPCM" [balPCMDesc_note]
+   [0 $<= sy time $<= sy time_final] [makeCite koothoor2013] eBalanceOnPCMDeriv 
+   "eBalanceOnPCM" balPCMDescNotes
 
 eBalanceOnPCM_rc :: RelationConcept
 eBalanceOnPCM_rc = makeRC "eBalanceOnPCM_rc" (nounPhraseSP
   "Energy Balance on PCM to find temperature of PCM")
   balPCMDesc balPCM_Rel -- eBalanceOnPCML
```
Comment (samm82) [inline code/drasil-example/Drasil/SWHS/IMods.hs]: Here, a completely different `Sentence` from the Notes is passed in. Should `RelationConcept` be taking a `Sentence` here? @JacquesCarette

File: code/drasil-example/Drasil/SWHS/IMods.hs [hunk 2]
```diff
@@ -204,7 +205,7 @@ eBalanceOnPCM = im eBalanceOnPCM_rc [qw temp_melt_P, qw time_final, qw temp_init
  qw pcm_HTC, qw pcm_mass, qw htCap_S_P, qw htCap_L_P]
   [sy temp_init $< sy temp_melt_P] (qw temp_PCM)
    [0 $<= sy time $<= sy time_final] [makeCite koothoor2013] eBalanceOnPCMDeriv 
-   "eBalanceOnPCM" [balPCMDesc_note]
+   "eBalanceOnPCM" balPCMDescNotes
 
 eBalanceOnPCM_rc :: RelationConcept
 eBalanceOnPCM_rc = makeRC "eBalanceOnPCM_rc" (nounPhraseSP
```
Comment (samm82) [inline code/drasil-example/Drasil/SWHS/IMods.hs]: In this `RelationConcept`, it takes in a completely different `Sentence` from the Notes - should `RelationConcept`s be taking these `Sentence`s? @JacquesCarette

**Comment by samm82** [comment]
@JacquesCarette That's just there so the Travis build works so this can be merged in, so that #1484 can focus on the missing DDs (so the PRs don't get too big). I made a note in #1484 to fix the referencing.

File: code/drasil-example/Drasil/SWHS/IMods.hs (hunk 1 above)
Comment (JacquesCarette) [reply code/drasil-example/Drasil/SWHS/IMods.hs]: I don't actually know... I'm not sure what either are supposed to mean, and where they are used.

**Comment by JacquesCarette** [comment]
So I should merge #1484 in first then?

**Comment by samm82** [comment]
#1484 is the issue for making the DDs, and I'm waiting for @smiths's approval on it. I figured I'd work on the IMs while I waited. @JacquesCarette

**Comment by JacquesCarette** [comment]
Working on these independently is fine, I still don't want to merge something in that puts weird stuff in stable -- they have this way of not going away in a reasonable time frame.

**Comment by samm82** [comment]
What should I do instead then?

**Comment by JacquesCarette** [comment]
Is that question about this issue in particular, or are you out of tasks in general?

**Comment by samm82** [comment]
About this PR - should I close it until the DDs are done and I can remove the FIXME?

**Comment by JacquesCarette** [comment]
Yes please.  But do keep merging master, else the conflicts will get out of hand.

**Comment by samm82** [comment]
Closed until DDs are done - don't forget to change the FIXME

**Pull request closed without merging**